Contributing/Issue_Priorities.md

```diff
@@ -0,0 +1,32 @@
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+## AMP GitHub Issue Priorities
+The AMP team is introducing the below priorities and guidelines to make it easier to order issues by the level of attention they need. 
```

Comment:
Since this documentation is intended for reference, I don't think it needs to refer to us "introducing" it or it being new.  We can send a separate announcement via the various channels once this is merged in.